### ISSUE TITLE
Hide the dark theme Option on Admin Site

### ISSUE THREAD

**Issue by makemypro** [unfoldadmin/django-unfold#302]
is there anyway so that i can hide the option of dark theme in admin site?

**Comment by erwaen** [comment]
Yes, you can override the template, specifically the `userlinks.html` file, where the button to change the theme (light, dark, system button) is located.

If you are not familiar with how to override templates in Django, there is a guide on this in the README of Unfold in the "Admin Dashboard Overriding Template" section.

Here are the steps to override the template:

1. In your project folder, create a `templates` folder.
2. Inside the `templates` folder, create an `unfold` folder to replace the files of Unfold you want.
3. For this case, the file you want to replace is located in the repository at the path `.../templates/unfold/helpers/userlinks.html`. So, create this same path in your project, and create the `userlinks.html` file. Copy the content from the repository like this:

    ```html
    {% load i18n static %}

    <div class="flex items-center ml-auto">
        <div class="flex flex-row group items-center leading-none relative">
            <div class="flex gap-4 items-center">
                {% if environment %}
                    {% include "unfold/helpers/label.html" with text=environment.0 type=environment.1 %}
                {% endif %}

                {% include "unfold/helpers/theme_switch.html" %}

                {% include "unfold/helpers/account_links.html" %}
            </div>
        </div>

        {% if nav_global_side %}
            {{ nav_global_side }}
        {% endif %}
    </div>
    ```

4. Remove the part where the theme switch button is included. In other words, delete this line: `{% include "unfold/helpers/theme_switch.html" %}`.

5. Lastly, if you haven't done it already, you need to configure your `settings.py` in your Django project to load your custom templates so that this file can definitely be overridden. Add the following to your `TEMPLATES` configuration:

    ```python
    TEMPLATES = [
        {
            "BACKEND": "django.template.backends.django.DjangoTemplates",
            "DIRS": [
                BASE_DIR / "templates",  # Make sure to use BASE_DIR for an absolute path
            ],
            "APP_DIRS": True,
            "OPTIONS": {
                "context_processors": [
                    "django.template.context_processors.debug",
                    "django.template.context_processors.request",
                    "django.contrib.auth.context_processors.auth",
                    "django.contrib.messages.context_processors.messages",
                ],
            },
        },
    ]
    ```

****
If for some reason what you really want is to just remove the dark theme option while still keeping the theme switch button, you have to edit the `theme_switch.html` file, which is located at `templates/unfold/helpers/theme_switch.html`. So, create this file with that path in your folder and copy the content:

```html
{% load i18n %}

<div class="relative" x-data="{ openTheme: false }">
    <a class="block cursor-pointer leading-none hover:text-gray-700 dark:text-gray-400 dark:hover:text-gray-200" x-on:click="openTheme = !openTheme">
        <span class="material-symbols-outlined">
            <span x-text="adminTheme == 'dark' && 'dark_mode' || adminTheme == 'light' && 'light_mode' || 'computer'"></span>
        </span>
    </a>

    <nav class="absolute bg-white border flex flex-col leading-none overflow-hidden py-1 -right-2 rounded shadow-lg text-sm text-gray-500 top-7 w-40 dark:bg-gray-800 dark:border-gray-700 dark:text-gray-400" x-cloak x-show="openTheme" @click.outside="openTheme = false">
        <a class="cursor-pointer flex flex-row leading-none mx-1 px-3 py-1.5 rounded hover:bg-gray-100 hover:text-gray-700 dark:hover:bg-gray-700 dark:hover:text-gray-200"
           x-on:click="adminTheme = 'dark'"
           x-bind:class="adminTheme == 'dark' && 'text-primary-600 dark:text-primary-500 dark:hover:!text-primary-500 hover:!text-primary-600'">
            <span class="material-symbols-outlined mr-2">dark_mode</span>
            <span class="leading-none self-center">{% trans "Dark" %}</span>
        </a>

        <a class="cursor-pointer flex flex-row mx-1 px-3 py-1.5 rounded hover:bg-gray-100 hover:text-gray-700 dark:hover:bg-gray-700 dark:hover:text-gray-200"
           x-on:click="adminTheme = 'light'"
           x-bind:class="adminTheme == 'light'  && 'text-primary-600 dark:text-primary-500 dark:hover:!text-primary-500 hover:!text-primary-600'">
            <span class="material-symbols-outlined mr-2">light_mode</span>
            <span class="leading-none self-center">{% trans "Light" %}</span>
        </a>

        <a class="cursor-pointer flex flex-row mx-1 px-3 py-1.5 rounded hover:bg-gray-100 hover:text-gray-700 dark:hover:bg-gray-700 dark:hover:text-gray-200"
           x-on:click="adminTheme = 'auto'"
           x-bind:class="adminTheme == 'auto' && 'text-primary-600 dark:text-primary-500 dark:hover:!text-primary-500 hover:!text-primary-600'">
            <span class="material-symbols-outlined mr-2">computer</span>
            <span class="leading-none self-center">{% trans "System" %}</span>
        </a>
    </nav>
</div>

```

Then, delete the <a> tag where the dark_mode is specified, meaning delete all this:

```html
<a class="cursor-pointer flex flex-row leading-none mx-1 px-3 py-1.5 rounded hover:bg-gray-100 hover:text-gray-700 dark:hover:bg-gray-700 dark:hover:text-gray-200"
           x-on:click="adminTheme = 'dark'"
           x-bind:class="adminTheme == 'dark' && 'text-primary-600 dark:text-primary-500 dark:hover:!text-primary-500 hover:!text-primary-600'">
            <span class="material-symbols-outlined mr-2">dark_mode</span>
            <span class="leading-none self-center">{% trans "Dark" %}</span>
        </a>
```

**Comment by lukasvinclav** [comment]
@makemypro Implemented in #308
